onnx_tf/handlers/backend/trilu.py

```diff
@@ -0,0 +1,35 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+
+
+@onnx_op("Trilu")
+class Trilu(BackendHandler):
+
+  @classmethod
+  def version_14(cls, node, **kwargs):
+    x = kwargs["tensor_dict"][node.inputs[0]]
+    k = tf.constant(0, dtype=tf.int32)
+    if len(node.inputs) >= 2:
+        k = kwargs["tensor_dict"][node.inputs[1]]
+    keep_triangle = tf.constant(-1, dtype=k.dtype)
+    upper = node.attrs.get("upper", 1)
+    
+    #handle pos out
+    if k > x.shape[-1]:
+        k = tf.constant(x.shape[-1], dtype=k.dtype)
+    elif k < 0-x.shape[-2]:
+        k = tf.constant(0-x.shape[-2], dtype=k.dtype)
```

Comment:
Please create a test case in (https://github.com/onnx/onnx-tensorflow/blob/master/test/backend/test_dynamic_shape.py) to ensure the shape based calculation works with dynamic input shapes